HouseRobber.java

```diff
@@ -0,0 +1,25 @@
+//Problem2 (https://leetcode.com/problems/house-robber/)
+//T: O(N)
+//S: O(N)
```

Comment:
Good Job, Please add the requested information as per the requested format along with an explanation
 of your approach

Reply:
Where is the requested format mentioned?